src/msf/mod.rs

```diff
@@ -11,10 +11,22 @@ use source::*;
 mod page_list;
 use self::page_list::PageList;
 
+use scroll::{Pread, LE};
 use std::fmt;
 
 type PageNumber = u32;
 
+/// The PDB header as stored on disk.
+#[derive(Debug, Pread)]
+struct RawHeader {
```

Comment:
I was going to suggest renaming `RawHeader` to match upstream, but [then I remembered why that's a bad idea](https://github.com/Microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/msf/msf.cpp#L946-L957). Still, this particular header format is specific to `BigMSF`, and I'd like to group those two together, either by naming or by tucking both together into a submodule.

Reply:
I have read entirely too much of that Microsoft PDB source but I still almost lost it at `PN32 mpspnpnSt`. :)

Reply:
I did this in a follow-up commit to make the initial diff easier to read since moving things into a submodule changes the indentation as well.